easybuild/easyconfigs/h/HBase/HBase-1.0.2.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PackedBinary'
+
+name = 'HBase'
+version = '1.0.2'
+
+homepage = 'http://hbase.apache.org/'
+description = """Apache HBase. is the Hadoop database, a distributed, scalable, big data store. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['%(namelower)s-%(version)s-bin.tar.gz']
```

Comment:
would be an option but not always necessary as it is basically Java code. Maybe we could make a distinction in the module name with a suffix? In case we want to add a built version from source?